sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed the request sent in `AzurePipelinesCredential` so it doesn't result in a redirect response when an invalid system access token is provided. 
 
 ### Other Changes
```

Comment:
Consider adding an entry in the "Other Changes" section, similar to https://github.com/Azure/azure-sdk-for-cpp/pull/6001

> - Allow certain response headers to be logged in `AzurePipelinesCredential` for diagnostics and include them in the exception message.